tools/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ add_clang_subdirectory(c-index-test)
 
 add_clang_subdirectory(clang-rename)
 
+add_clang_subdirectory(clang-display-context)
```

Comment:
I think LLVM is at least somewhat enforcing an alphabetical order.

Reply:
It is, but in this particular file it has been already broken, so I have just appended to the end of the list.